server/lib/update_user_database.js

```diff
@@ -0,0 +1,46 @@
+// import { getUserDatabase } from './get_user_database.js';
+
+const pgClient = require('./pg_client');
+const getUserDatabase = require('./get_user_database').getUserDatabase;
+
+const updateUserDatabase = (connection, userId, updates, callback) => {
+  const client = pgClient(connection);
+  getUserDatabase(connection, userId, (getUserErr, getUserReply) => {
+    if (getUserErr) {
+      callback(getUserErr);
+    } else {
+      const user = getUserReply.rows[0];
+      const updateFields = Object.keys(updates);
+      updateFields.forEach((item, index) => {
+        user[item] = updates[updateFields[index]];
+      });
+      const userIdToUpdate = user.id;
```

Comment:
I don't know what that is and I couldn't figure it out within 30 seconds. Probably shouldn't be in the project.